submissions/RemoveBG_Extension/manifest.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "CopyBG",
+  "version": "1.0.0",
+  "manifest_version": 2,
```

Comment:
When I try loading the extension in Chrome it now shows a warning dialog telling me the extension will soon not work. 

I'm sorry this is taking so long to review but please can you look into this? It has something to do with the [manifest version 2 getting deprecated](https://developer.chrome.com/docs/extensions/develop/migrate/mv2-deprecation-timeline#:~:text=We%20will%20now%20begin%20disabling%20installed%20extensions%20still%20using%20Manifest%20V2%20in%20Chrome%20stable.)